spot-ingest/install.sh

```diff
@@ -0,0 +1,161 @@
+        fi  
+
+        if [[ "${missing_dep[@]}" ]]; then
+                log_cmd "installing ${missing_dep[@]}"
+                ${install_cmd} ${missing_dep[@]}
+                unset missing_dep[*]
+        fi  
+}
+
+check_tshark () {
+        # check dependencies only, installs in custom location
+        log_cmd "installing dependencies for tshark installation"
+        if [ "${host_os}" == "debian" ]; then
+                check_bin make bzip2 pkg-config libsmi flex bison byacc
+                install_pkg libpcap-dev heimdal-dev libc-ares-dev
+        elif [ "${host_os}" == "rhel" ]; then
+                check_bin make bzip2 gcc bison
+                install_pkg glib2-devel flex-devel libsmi-devel libpcap-devel
+        fi
+}
+
+install_tshark () {
+        if type tshark >/dev/null 2>&1; then
+                log_cmd "tshark found"
+        else
+                log_cmd "tshark missing"
+                check_tshark
+                ${wget_cmd} https://1.na.dl.wireshark.org/src/wireshark-${wshark_vers}.tar.bz2 -P ${source_path}/
+                ${untar_cmd} ${source_path}/wireshark-${wshark_vers}.tar.bz2 -C ${source_path}/
+                cd ${source_path}/wireshark-${wshark_vers}
+                log_cmd "compiling tshark"
+                ./configure --prefix=${install_path} --enable-wireshark=no
+                make ${mk_opt} 
+                make install
+                cd ..
+        fi
+        log_cmd "tshark build complete"
+        tshark -v
+}
+
+install_nfdump () {
+        if type nfdump >/dev/null 2>&1; then
+                log_cmd "nfdump found"
+        else
+                log_cmd "installing spot-nfdump"
+                ${wget_cmd} https://github.com/Open-Network-Insight/spot-nfdump/archive/${nfdump_vers}.tar.gz -P ${source_path}/
+                ${untar_cmd} ${source_path}/${nfdump_vers}.tar.gz -C ${source_path}/
+                cd ${source_path}/spot-nfdump-*/
+                source ./install_nfdump.sh ${install_path}
+                cd ${local_path}    
+        fi
+}
+
+install_pip () {
+        if type pip >/dev/null 2>&1; then\
+                log_cmd "pip found"
+        else    
+                log_cmd "missing pip"
+                ${wget_cmd} https://bootstrap.pypa.io/get-pip.py -P ${source_path}/
+                python ${source_path}/get-pip.py
+                log_cmd "pip installed"
+        fi
+}
+
+# end functions
+
+check_root
+check_os
+
+if [ ! -d ${source_path} ]; then
+        mkdir ${source_path}
+fi
+
+if [ ! -d ${install_path} ]; then
+        log_cmd "${install_path} not created, Please run spot-setup/install.sh first"
+        exit 1        
+fi
+
+# check basic dependencies
+check_bin ${dependencies[@]}
+install_pkg
+
+# install dissectors
+install_tshark
+install_nfdump
+
+# python dependencies
+install_pip
+if [ -z ${local_path}/requirements.txt ]; then
+    pip install -r requirements.txt
+fi
+
+log_cmd "dependencies satisfied, please run ./build.sh to complete setup"
```

Comment:
spot-ingest install.sh is asking to run build.sh but there is no spot-ingest/build.sh, is this message something should be here or build.sh is missing?